cirq/circuits/circuit.py

```diff
@@ -1527,10 +1530,23 @@ def _get_operation_circuit_diagram_info_with_fallback(
 
     return protocols.CircuitDiagramInfo(wire_symbols=symbols)
 
+def _is_valid_identifier(text):
+    return re.match('[a-zA-Z_][a-zA-Z0-9_]*$', text)
 
-def _formatted_exponent(info: protocols.CircuitDiagramInfo,
+
+def _encode(text):
+    return json.JSONEncoder().encode(text)
+
+def _is_exposed_formula(info: protocols.CircuitDiagramInfo,
```

Comment:
Rename back to `_formatted_exponent`. Methods with `_is` at the start of their name should return booleans.

Reply:
Done.